taqueria-types.ts

```diff
@@ -1,3 +1,4 @@
+debugger;
```

Comment:
Do we want this be merged?

Reply:
Good find. I've removed that one, as well as a couple of others!. Thanks Ali!